apps/api-gateway/src/board/board.controller.ts

```diff
@@ -52,54 +42,82 @@ export class BoardController {
     description: '게시글 상세 조회',
     example: {
       post_id: 1,
-      title: '게시물 제목',
-      content: '게시물 내용',
-      update_at: '2024-10-01T00:00:12',
-      create_at: '2024-10-03T00:00:12',
-      expired_at: '2024-10-06T00:00:12',
-      view_cnt: 15,
-      bookmarked_cnt: 3,
+      title: '프로젝트명',
+      content: '프로젝트 내용',
+      create_at: '2024-10-08T06:16:30.000Z',
+      expired_at: '2024-10-12T00:00:00.000Z',
+      view_cnt: 0,
+      bookmarked_cnt: 0,
       is_open: true,
+      project_category_id: ['3'],
+      stack_category_id: ['1', '4'],
+      post_category_id: {
+        id: 2,
+        post_category_name: '그랩존',
+      },
+      career_category_id: {
+        career_category_id: 1,
+        content: '0년차',
+      },
+      position_category_id: null,
     },
   })
   public async getPostById(@Param('id') id: number) {
     return await this.boardService.getPostById(id);
   }
 
-  @Post()
+  @Post('category/:postCategoryId')
   @ApiOperation({ summary: '게시글 생성' })
   @ApiCreatedResponse({
     description: '게시글 생성',
     example: {
-      post_id: 1,
-      title: '게시물 제목',
-      content: '게시물 내용',
-      update_at: '2024-10-01T00:00:12',
-      create_at: '2024-10-03T00:00:12',
-      expired_at: '2024-10-06T00:00:12',
-      view_cnt: 15,
-      bookmarked_cnt: 3,
+      title: '프로젝트명',
+      content: '프로젝트 소개',
+      expired_at: '2024-10-10T00:00:00.000Z',
+      project_category_id: [2, 4],
+      stack_category_id: [4, 5],
+      post_category_id: {
+        id: 2,
+      },
+      career_category_id: null,
+      position_category_id: null,
+      post_id: 15,
+      create_at: '2024-10-08T23:23:46.000Z',
+      update_at: '2024-10-08T23:23:46.000Z',
+      view_cnt: 0,
+      bookmarked_cnt: 0,
```

Comment:
```suggestion
      title: '프로젝트명',
      content: '프로젝트 소개',
      expired_at: '2024-10-10T00:00:00.000Z',
      project_category_id: [2, 4],
      stack_category_id: [4, 5],
      post_category_id: 2,
      career_category_id: null,
      position_category_id: null,
      post_id: 15,
      create_at: '2024-10-08T23:23:46.000Z',
      update_at: '2024-10-08T23:23:46.000Z',
      view_cnt: 0,
      bookmarked_cnt: 0,
```

POST Request 시에는 해당 데이터에 int 형만 들어가는 방향이 맞지 않을까 싶습니다!

Reply:
수정했습니다 감사합니다! 
ef0c006fbb152134a3fffc5af4949f7843f1bf8c